notebooks/week2/05_log_and_register_fe_model.py

```
@@ -0,0 +1,280 @@
+dbutils.library.restartPython()
+
+# COMMAND ----------
+
+import yaml
+import mlflow
+from mlflow.models import infer_signature
+
+from datetime import datetime
+
+from pyspark.sql import SparkSession
+import pyspark.sql.functions as F
```

Ruff

Module level import not at top of file

(E402)